compiler/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "vyper-compiler"
+version = "0.1.0"
+authors = ["Grant Wuerker <gwuerker@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+vyper-parser = {path = "../parser", version = "0.1.0"}
+serde_json = "1.0"
+serde = "1.0"
+hex = "0.4"
+yultsur = { git = "https://github.com/g-r-a-n-t/yultsur" }
+solc = { git = "https://github.com/g-r-a-n-t/solc-rust" }
```

Comment:
anything specific in this fork?

Reply:
I needed to update the version of solc that it binds to to get Yul compilation.

https://github.com/axic/solc-rust/pull/1

Reply:
cool, looks good; we will want to switch to the upstream when we can but that can happen in a later PR